service/src/main/java/bio/terra/policy/db/PaoDao.java

```diff
@@ -158,7 +158,7 @@ public Pao getPao(UUID objectId) {
       Map<String, PolicyInputs> attributeSetMap =
           getAttributeSets(List.of(dbPao.attributeSetId(), dbPao.effectiveSetId()));
       return Pao.fromDb(dbPao, attributeSetMap);
-    } catch (EmptyResultDataAccessException e) {
```

Comment:
Instead of catching a `InternalTpsErrorException` here, could `getDbPao` throw a `EmptyResultDataAccessException` if the Pao is not found? that seems like a more specific (and accurate) exception